src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -179,6 +179,26 @@ export class GuidedTourService {
         return false;
     }
 
+    /**
+     * Determines if the cancel tour is currently displayed
+     */
+    public isCancelTourStep(): boolean {
+        if (this.currentTour && this.currentTour.steps) {
+            return this.currentTour.settingsKey === cancelTour.settingsKey;
+        }
+        return false;
+    }
+
+    /**
+     * Determines if the completed tour is currently displayed
+     */
+    public isCompleteTourStep(): boolean {
+        if (this.currentTour && this.currentTour.steps) {
+            return this.currentTour.settingsKey === completedTour.settingsKey;
+        }
+        return false;
+    }
```

Comment:
You could merge those into one method having the settings key as a parameter

Reply:
@ungaralex I had the implementation just like suggested but then I thought that having the method names `isCancelTourStep()` and `isCompleteTourStep()` is easier to read :)